client/src/components/Earn/EarnBoxPersonalAllocation/EarnBoxPersonalAllocation.tsx

```diff
@@ -91,9 +91,7 @@ const EarnBoxPersonalAllocation: FC<EarnBoxPersonalAllocationProps> = ({ classNa
         coinPricesServerDowntimeText: !isProjectAdminMode ? '...' : undefined,
         cryptoCurrency: 'ethereum',
         dataTest: 'BoxPersonalAllocation__Section--availableNow__DoubleValue',
-        isFetching: isPatronMode
-          ? isFetchingTotalPatronDonations
-          : isFetchingWithdrawals || isAppWaitingForTransactionToBeIndexed,
+        isFetching: isPatronMode ? isFetchingTotalPatronDonations : isFetchingWithdrawals,
```

Comment:
It's not that simple, unfortunately. Because when you withdraw ETH, there needs to be a skeleton here.

You need to distinguish between transactionPending that comes from lock/unlock and the one that comes from withdrawing.